README.md

```diff
@@ -6,45 +6,69 @@ A Floating UI wrapper for Svelte.
+The `useFloating` hook takes care of positioning your floating UI elements (tooltips, popovers, etc.) relative to another element.
+It automatically calculates the best placement and updates it as needed, giving you properties to access the position and styles.
 
-To publish your library to [npm](https://www.npmjs.com):
+#### Import
 
-```bash
-npm publish
+```js
+import { useFloating } from '@skeletonlabs/floating-ui-svelte';
 ```
+
+#### Options
+
+| Property             | Description                                                                                                                      | Type                                                                                       | Default Value |
+| -------------------- | -------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ | ------------- |
+| open                 | Represents the open/close state of the floating element.                                                                         | boolean                                                                                    | true          |
+| onOpenChange         | Event handler that can be invoked whenever the open state changes.                                                               | (open: boolean, event?: Event, reason?: OpenChangeReason) => void                          | -             |
+| placement            | Where to place the floating element relative to its reference element.                                                           | Placement                                                                                  | 'bottom'      |
+| strategy             | The type of CSS position property to use.                                                                                        | Strategy                                                                                   | 'absolute'    |
+| middleware           | These are plain objects that modify the positioning coordinates in some fashion, or provide useful data for the consumer to use. | Array<Middleware \| undefined \| null \| false>                                            | undefined     |
+| transform            | Whether to use `transform` instead of `top` and `left` styles to position the floating element (`floatingStyles`).               | boolean                                                                                    | true          |
+| elements             | The reference and floating elements.                                                                                             | FloatingElements                                                                           | -             |
```

Comment:
I guess `The reference and floating elements.` should be `The reference **to the** floating elements.` or just `**The** floating elements.`

Reply:
IMO the name of the variable speaks for itself, on top of the example above. Though I'll leave this open for others to comment about.

Reply:
I prefer `The floating elements.`